dev-utils/src/dev_utils.rs

```diff
@@ -19,12 +20,20 @@ pub enum DevUtils {
         #[structopt(subcommand)]
         genesis_command: GenesisCommand,
     },
+
+    /// Used for initializing
```

Comment:
I think this is obvious from the name (it'd be surprising if "init" wasn't for "initializing") -- it'd be more helpful if the comment (and `about = "..."` that's printed in the "help" of CLI) would describe what exactly it is doing in high-level terms: e.g. creates a few default wallets (that can then be used with client-cli), generates a genesis state that contains addresses corresponding to these wallets, and copies it into Tendermint's folder (I may have forgotten something)

Reply:
ok, add more detail